package.json

```diff
@@ -86,6 +86,7 @@
     "jsdom": "24.0.0",
     "markdownlint-cli": "0.39.0",
     "postcss-html": "1.6.0",
+    "postcss-nesting": "12.1.0",
```

Comment:
Move it it to `dependencies` please, everything that the build depends on should be there:

```
npm uninstall postcss-nesting && npm i postcss-nesting